{"uuid": "0567c680-626d-4a1d-af25-3a0c41fe4636", "children": ["ca707aa0-1d25-47c3-b461-eea3be2dd03e"], "befores": [{"name": "init_driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 112\nCurrent browser version is 122.0.6261.70 with binary path C:\\Users\\pappula.l\\AppData\\Local\\Google\\Chrome\\Application\\chrome.exe\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x004DDCE3+50899]\n\t(No symbol) [0x0046E111]\n\t(No symbol) [0x00375588]\n\t(No symbol) [0x0039570C]\n\t(No symbol) [0x00391471]\n\t(No symbol) [0x0038F479]\n\t(No symbol) [0x003C1FFE]\n\t(No symbol) [0x003C1CEC]\n\t(No symbol) [0x003BB6F6]\n\t(No symbol) [0x00397708]\n\t(No symbol) [0x0039886D]\n\tGetHandleVerifier [0x00743EAE+2566302]\n\tGetHandleVerifier [0x007792B1+2784417]\n\tGetHandleVerifier [0x0077327C+2759788]\n\tGetHandleVerifier [0x00575740+672048]\n\t(No symbol) [0x00478872]\n\t(No symbol) [0x004741C8]\n\t(No symbol) [0x004742AB]\n\t(No symbol) [0x004671B7]\n\tBaseThreadInitThunk [0x761C6839+25]\n\tRtlGetFullPathName_UEx [0x774B901F+1215]\n\tRtlGetFullPathName_UEx [0x774B8FED+1165]\n\n", "trace": "  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\GIT_Projects\\CAW\\tests\\conftest.py\", line 15, in init_driver\n    web_driver = webdriver.Chrome(service=service_obj)\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 49, in __init__\n    super().__init__(\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 54, in __init__\n    super().__init__(\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 206, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 291, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\pappula.l\\AppData\\Roaming\\Python\\Python310\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1709057745697, "stop": 1709057754046}], "start": 1709057745697, "stop": 1709057754538}